Source/CombatExtended/CombatExtended/Lasers/LaserBeamCE.cs

```diff
@@ -11,11 +11,11 @@ namespace CombatExtended.Lasers
     public class LaserBeamCE : BulletCE
     {
 
-	public override float DamageAmount
+        public override int DamageAmount
```

Comment:
Why is this being converted *back* to an int?  We'll likely be moving all bullet damage to floats eventually, no reason to add more work later for lasers.